datacenter/ucp/2.0/guides/release-notes.md

```diff
@@ -10,6 +10,34 @@ known issues for the latest UCP version.
 You can then use [the upgrade instructions](installation/upgrade.md), to
 upgrade your installation to the latest release.
 
+## Version 2.0.3
+
+(8 Feb 2017)
+
+**Bug Fixes**
+
+* Core
+  * Added role-based access control for volumes
```

Comment:
Here's what we used in UCP 2.1 beta:
* Label-based access control now supported for volumes.
	(NOTE: unlike other resources controlled via label-based access control, a 
	volume without a label is accessible by all UCP users with Restricted Control 
	or higher default permissions.)